shared/constants/form.ts

```diff
@@ -92,6 +92,9 @@ export const PAYMENT_VARIABLE_INPUT_AMOUNT_FIELD_ID =
 export const CLIENT_RADIO_OTHERS_INPUT_VALUE =
   '!!FORMSG_INTERNAL_RADIO_OTHERS_VALUE!!'
 
+export const CLIENT_CHECKBOX_OTHERS_INPUT_VALUE =
```

Comment:
This should be in the MRF BE validation PR?

Reply:
yes it is, will resolve the merge conflict when merging into dev.

Reply:
need to rebase depending on if BE validation PR or this goes into master first

Reply:
just squashed and merged BE val, will rebase now

Reply:
resolved